.env.local

```diff
@@ -0,0 +1,6 @@
+TESTING_MODE=true
+APPINSIGHTS_INSTRUMENTATIONKEY=2f1d6a9c-1b88-4f3d-bba1-64b11ffd2362
+GET_SUBMISSIONS_URL=https://dddperth-functions-test.azurewebsites.net/api/GetSubmissions
+SUBMIT_VOTE_URL=https://dddperth-functions-test.azurewebsites.net/api/SubmitVote
+GET_AGENDA_URL=https://dddperth-functions-test.azurewebsites.net/api/GetAgenda
+SUBMIT_FEEDBACK_URL=https://dddperth-functions-test.azurewebsites.net/api/SubmitFeedback
```

Comment:
We don't push these config mate :)

Reply:
we have been, this is just moved from package.json to here